### DISCUSSION TITLE
ECHAM6, how to output atmospheric humidity at the lowest model level?

### DISCUSSION THREAD

**Discussion by QinggangGao** [esm-tools/esm_tools#1066] [General]
Dear community

May I ask, is there a way to output three-dimensional atmospheric humidity only at the lowest model level, so as to save storage space?

Regards, Qinggang

**Comment by seb-wahl** [comment]
Yes, I think this is possible. Couldn't you use 2m humidity. It is available as an output parameter (`q2m` or `rh2m`).
I think you can also only select certain levels but I would need to read the users guide for this as well.

**Comment by QinggangGao** [comment]
Hi, thank you. I know there are q2m and rh2m, but I indeed need atmospheric humidity of water isotopes, which does not have default output of wisorh2m nor wisoq2m. So I would like to have a general method to output certain levels. I will check the user guide, and please let me know if you know how to do it with esm_tools.

Regards, Qinggang
